packages/darklang/stdlib/uint128.dark

```diff
@@ -0,0 +1,115 @@
+module Darklang =
+  module Stdlib =
+    module UInt128 =
+
+      type ParseError =
+        | BadFormat
+        | OutOfRange
+
+
+      /// Returns the result of wrapping <param a> around so that {{0 <= res < b}}.
+      /// The modulus <param b> must be greater than 0.
+      /// Use <fn UInt128.remainder> if you want the remainder after division, which has
+      /// a different behavior for negative numbers.
+      let ``mod`` (a: UInt128) (b: UInt128) : UInt128 = Builtin.UInt128.``mod`` a b
+
+
+      /// Adds two 128-bit unsigned integers together
+      let add (a: UInt128) (b: UInt128) : UInt128 = Builtin.UInt128.add a b
+
+
+      /// Subtracts two 128-bit unsigned integers
+      let subtract (a: UInt128) (b: UInt128) : UInt128 = Builtin.UInt128.subtract a b
+
+
+      /// Multiplies two 128-bit unsigned integers
+      let multiply (a: UInt128) (b: UInt128) : UInt128 = Builtin.UInt128.multiply a b
+
+
+      /// Divides two 128-bit unsigned integers
+      let divide (a: UInt128) (b: UInt128) : UInt128 = Builtin.UInt128.divide a b
+
```

Comment:
Did you remove absoluteValue?

Reply:
I haven't implemented an absoluteValue function for unsigned integers. Using it on a non-negative number would have no effect